assets/chat/js/hashlinkconverter.js

```diff
@@ -13,8 +13,8 @@ class HashLinkConverter {
     this.rumbleEmbedRegex = /^embed\/([a-z0-9]+)\/?$/;
   }
 
-  convert(urlString) {
-    if (!urlString) {
+  convert(urlString, throwErrors) {
```

Comment:
I'd rather callers who don't care about exceptions simply catch and ignore. I think that's a better approach.

Reply:
Agreed